src/blockchain/log-processors/index.ts

```diff
@@ -36,7 +48,7 @@ export const logProcessors: LogProcessors = {
       add: processInitialReporterRedeemedLog,
       remove: processInitialReporterRedeemedLogRemoval,
     },
-    InitialReporterTransferred: {
+    InitialReporterTransfered: {
```

Comment:
I think this should be `InitialReporterTransferred` shouldn't it? The typo of one `r` was fixed in augur core.

Reply:
Yea i noticed that... hmmm, i must have an old version checked out? I had to make this change to make it work, did this one change after the other ones?

Reply:
There hasn't been an augur-core release since -18, which was before you changed that name.